Gemfile

```diff
@@ -5,11 +5,9 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-spree_version = '>= 4.5'
+spree_version = '>= 5.0.3'
```

Comment:
we are already testing 5.1.0.beta
so lets keep this to 5.2.0 for now instead of 5.0.3